target/arc/mmu-v6.c

```diff
@@ -713,7 +713,12 @@ void arc_mmu_init_v6(CPUARCState *env)
         }
         break;
     case ARC_OPCODE_ARC32:
-        mmu_v6_version = &mmuv6_info[MMUV6_32_4K];
+        if(cpu->cfg.mmuv6_version == NULL ||
```

Comment:
space between `if` and `(`. Don't forget to align the following line accordingly after editing this line.